assembl/static2/js/app/graphql/mutations/createResource.graphql

```diff
@@ -0,0 +1,22 @@
+mutation createResource(
+  $doc: String
+  $embedCode: String
+  $image: String
+  $textEntries: [LangStringEntryInput]
+  $titleEntries: [LangStringEntryInput]!
```

Comment:
@cedricmessiant fyi if you want the list to have at least one element, I think you can write`[LangStringEntryInput!]!`

Reply:
done ! note that we can't do this easily for `textEntries` (I think it is because it creates `<p>` tags)